### PR TITLE
Remove tf.contrib detail from the document.

### PR DESCRIPTION
**Pull request by sachinprasadhs**
Removed tf.contrib detail from the tf.compat.v1.estimator.tpu.TPUEstimatorSpec document which would create confusion to the community.

### CODE REVIEW

**Review by mihaimaruseac — APPROVED** [review]
Approving, but note that Estimator is no longer active, currently it's only updated as needed to fully deprecate from TF.

**Review by nfelt — APPROVED** [review]

**Comment by mihaimaruseac** [comment]
> Instead of removing the sentence, replacing this with `tf.summary.create_file_writer`?

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]
Instead of removing the sentence, replacing this with `tf.summary.create_file_writer`?

File: tensorflow_estimator/python/estimator/tpu/tpu_estimator.py [hunk 1]
```diff
@@ -310,8 +310,8 @@ class TPUEstimatorSpec(model_fn_lib._TPUEstimatorSpec):  # pylint: disable=prote
   executed on the CPU on every step, so there is communication overhead when
   sending tensors from TPU to CPU. To reduce the overhead, try reducing the
   size of the tensors. The `tensors` are concatenated along their major (batch)
-  dimension, and so must be >= rank 1. The `host_call` is useful for writing
-  summaries with `tf.contrib.summary.create_file_writer`.
+  dimension, and so must be >= rank 1.  The `host_call` is useful for writing
```
Comment (mihaimaruseac) [inline tensorflow_estimator/python/estimator/tpu/tpu_estimator.py:313]:
```suggestion
  dimension, and so must be >= rank 1. The `host_call` is useful for writing
```

**Pull request merged**